### DISCUSSION TITLE
Add scan QR option

### DISCUSSION THREAD

**Discussion by arcturiano** [BitCannaGlobal/WeedMobile#92] [Ideas]
I  am trying to send some BCNA to Raul via the QR code that he shared in weedmobile-beta-testing channel in Discord

1. I click on SEND button in Dashboard inside app
2. I see the Send Token screen, but there is no way to scan QR code in there

I suggest adding a button beside the "address book" button that enables QR scanning and detect the address

Other mobile wallets have such an option and would make it easier to send tokens in my opinion. Not needing to add a contact

![sendbutton](https://github.com/BitCannaGlobal/WeedMobile/assets/94879750/2d97aa3b-185d-42e6-9e81-ab84cf5ed39a)
![sendtokens](https://github.com/BitCannaGlobal/WeedMobile/assets/94879750/ef28097d-9870-4cc0-9ad6-87a3a09b4842)

**Comment by TJ-BitCanna** [comment]
Thanks for putting this up!

Do you intend to only be able to scan a QR to add the address?
Or more to be able to go from the "Send" screen to the "Scan QR" page without needing to go back to the Dashboard page?

**Comment by arcturiano** [comment]
I was thinking more of being able to add address by scanning QR code, so it is added there and to contacts. Raul already explained that this would be used more for paying and ecommerce, which makes sense, but what if someone wants to send a friend some BCNA? Or to onboard someone to the network.

**Reply by TJ-BitCanna** [reply]
Clear!

I'll mark it as an enhancement ^^
